CMakeLists.txt

```diff
@@ -83,11 +113,43 @@ add_compile_definitions(PAGER_ENV="LESS=FRX LV=-c"
 						BINDIR="bin"
 						GIT_BUILT_FROM_COMMIT="")
 
-set(FALLBACK_RUNTIME_PREFIX /home/$ENV{USER})
-add_compile_definitions(FALLBACK_RUNTIME_PREFIX="${FALLBACK_RUNTIME_PREFIX}")
+if(WIN32)
+	set(FALLBACK_RUNTIME_PREFIX /mingw64)
+	add_compile_definitions(FALLBACK_RUNTIME_PREFIX="${FALLBACK_RUNTIME_PREFIX}")
+else()
+	set(FALLBACK_RUNTIME_PREFIX /home/$ENV{USER})
+	add_compile_definitions(FALLBACK_RUNTIME_PREFIX="${FALLBACK_RUNTIME_PREFIX}")
+endif()
+
+
+#Platform Specific
+if(${CMAKE_SYSTEM_NAME} STREQUAL "Windows")
```

Comment:
Why is this not `if(WIN32)` like the other Windows-specific blocks?

Reply:
This is because cmake considers both apple and linux to be UNIX, which means there will be a nested if block for apple and linux. By doing this we can avoid that, plus I think this is easier to maintain.

Reply:
Wouldn't CMake consider Cygwin also to match this, but not `WIN32`? I ask because we would need a Cygwin-specific block here, and the current settings would be incorrect for Cygwin.